### DISCUSSION TITLE
PathChooserInput does not detect windows mapped network drive?

### DISCUSSION THREAD

**Discussion by ghost** [alejandroautalan/pygubu-designer#257] [Q&A] [answered]
I'm trying to specify a mapped network drive (Windows) for the PathChooserInput widget, but it does not appear in my list of directories. Is this a documented behavior?

**Comment by alejandroautalan** [comment, marked as answer]
Hello @pyAnsysUser, thanks for trying pygubu.
PathChooserInput uses the filedialog module functions.

Can you post some screenshots of the issue and the expected result?

Regards
Alejandro A.

**Reply by ghost** [reply]
let me try some more searching on filedialog module expected behavior. This only occurs on one machine of three in test. I do not expect I will be able to post screenshots of the issue and expected result.
